src/app/components/OrderInput.tsx

```diff
@@ -479,7 +541,15 @@ function UserInputContainer() {
         <>
           <CurrencyInputGroup userAction={UserAction.UPDATE_PRICE} />
           <CurrencyInputGroup userAction={UserAction.SET_TOKEN_1} />
-          <PercentageSlider />
```

Comment:
The percentage slider in market orders should be **below** the input field, not above.
<img width="340" alt="Bildschirmfoto 2024-09-19 um 04 35 02" src="https://github.com/user-attachments/assets/4e576e77-ed9e-4e7d-9454-a65762847ce7">